css/image/cloud-sync-service-amd64/Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM alpine:3.6
 
-RUN apk --no-cache add libcrypto1.0 libssl1.0 ca-certificates 
+RUN addgroup -g 1000 -S cssuser && adduser -u 1000 -S cssuser -G cssuser \
+        && apk --no-cache add libcrypto1.0 libssl1.0 ca-certificates
 
```

Comment:
Need to update the anax/Makefile with the new version/tag of the CSS if you want the build system to create a new container.

Reply:
The make file has been updated to css version 1.0.11.